### PR TITLE
chore: ini ethers rev

### PR DESCRIPTION
**Pull request by shunkakinoki**
Pin the `ethers-solc` dep

### CODE REVIEW

**Review by Vid201 — CHANGES_REQUESTED** [review]
Hey, thanks for the PR. Make sense to change that, but I am not sure the best place for ethers-solc is in the workspace Cargo.toml since it's only used in the crate contracts. Let's change there the version and fix the conflicts - then it's good to merge!

**Comment by shunkakinoki** [comment]
Updated! Thank you so much 🙌

File: crates/contracts/Cargo.toml [hunk 1]
```diff
@@ -26,4 +26,4 @@ tokio = { workspace = true }
 [build-dependencies]
 anyhow = "1"
 ethers = { workspace = true, features = ["solc-full"] }
-ethers-solc = "2.0.8"
+ethers-solc = { git = "https://github.com/gakonst/ethers-rs", rev = "fa3017715a298728d9fb341933818a5d0d84c2dc" }
```
Comment (zsluedem) [inline crates/contracts/Cargo.toml:29]: Could you please give more info on why 2.0.8 not working and what you are fixing? I don't understand stand the intention here

**Review by zsluedem — CHANGES_REQUESTED** [review]

File: crates/contracts/Cargo.toml (hunk 1 above)
Comment (shunkakinoki) [reply crates/contracts/Cargo.toml:29]: Thank you for the review!

In my local machine, `2.0.9` introduces breaking changes and does not compile - also to keep in sync w/ the workspace version!

Comment (zsluedem) [reply crates/contracts/Cargo.toml:29]: Would you mind post the compile error here?

**Pull request closed without merging**